hr_timesheet_default_analytic_account/readme/INSTALL.rst

```diff
@@ -0,0 +1 @@
+To install this module, you need to have the `hr_timesheet_sheet <https://github.com/OCA/timesheet/tree/12.0/hr_timesheet_sheet>`_ OCA module available.
```

Comment:
Il faut renseigner les dépendances à des repository externes dans le fichier oca_dependencies.txt à la racine du repo. [Exemple](https://github.com/OCA/sale-workflow/blob/14.0/oca_dependencies.txt).

Reply:
thanks, good to know! should i leave the `INSTALL.rst` nevertheless or is it useless?

Reply:
the `timesheet` oca project is already listed in [`oca_dependencies.txt`](https://github.com/coopiteasy/cie-timesheet/blob/12.0-mig-hr_timesheet_default_analytic_account/oca_dependencies.txt). 👍